### PR TITLE
add explore page mobile view

### PR DESCRIPTION
**Pull request by and-gurin**

### CODE REVIEW

File: src/components/searchPanel/searchPanel.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.search-panel {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    padding: 0 12px;
+    max-width: 1073px;
+    width: 100%;
+    height: 54px;
```
Comment (zumbost1k) [inline src/components/searchPanel/searchPanel.css:9]: styles that determine the size should not be in the block, but in the element that is signed like this relative-block__block

File: src/components/searchPanel/searchPanel.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+.search-panel__input {
+    margin-left: 18px;
+}
+
+.search-panel__button {
+    display: block;
+    color: #FFF;
+    font-size: 12px;
+    font-style: normal;
```
Comment (zumbost1k) [inline src/components/searchPanel/searchPanel.css:23]: display: block;
font-style: normal;
are not needed. They are standard styles that do not need to be set.

File: src/components/searchPanel/searchPanel.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Search from '../../icons/search';
```
Comment (zumbost1k) [inline src/components/searchPanel/searchPanel.jsx:2]: use '@/icons/search'; instead

File: src/components/searchPanel/searchPanel.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import './searchPanel.css'
+import CustomButton from "../../UI/button/button";
```
Comment (zumbost1k) [inline src/components/searchPanel/searchPanel.jsx:4]: use @ instead of dots. do that everywhere.

**Review by zumbost1k — COMMENTED** [review]
Also make changes to your commit according to the changes to my file after the review. I changed the file names of the custom button and added props to the filer

File: src/components/searchPanel/searchPanel.css (hunk 1 above)
Comment (and-gurin) [reply src/components/searchPanel/searchPanel.css:9]: Do you mean I need to make an extra container with element dimensions?

Comment (zumbost1k) [reply src/components/searchPanel/searchPanel.css:9]: yes. add and add size characteristics inside of it 
relative-block__block{ margin: 0 auto;
    padding: 0 12px;
    max-width: 1073px;}

**Pull request merged**